### ISSUE TITLE
TeamCity config (Jmeter_plugin)

### ISSUE THREAD

**Issue by Ronski006** [jtorgan/jmeter_plugin#12]
Hi,

I am unable to locate "Performance Metrics Calculation" within Build Configuration Settings || Build Features.

Screenshots have been attached for your information.
[TeamCity_config.docx](https://github.com/jtorgan/jmeter_plugin/files/123136/TeamCity_config.docx)

Your assistance is appreciated.

**Comment by BenWaller** [comment]
Yeah, I'm not seeing the build features this plugin is supposed to add. It could be that it doesn't work with Teamcity 9.x
